roles/container_runtime/templates/daemon.json

```diff
@@ -5,10 +5,15 @@
     "disable-legacy-registry": false,
     "exec-opts": ["native.cgroupdriver=systemd"],
     "insecure-registries": {{ l_docker_insecure_registries }},
-{% if openshift_docker_log_driver  %}
-    "log-driver": "{{ openshift_docker_log_driver }}",
-{%- endif %}
-    "log-opts": {{ l_docker_log_options_dict }},
+    {% if openshift_docker_log_driver  %}
+        "log-driver": "{{ openshift_docker_log_driver }}",
+    {%- endif %}
+    {% if 'max-size' in l_docker_log_options_dict | from_json -%}
```

Comment:
Oops...this is the place where we allow user-specified 'max-size' to override the default.